src/services/telemetryService.js

```diff
@@ -25,12 +31,37 @@ export class TelemetryService {
   /** Returns a list of active alarms (open or ack) */
   static getActiveAlarms(params = {}) {
     return HttpClient.get(`${ENDPOINT}alarmsbyrule?${stringify(params)}`)
-      .map(toActicveAlarmsModel);
+      .map(toActiveAlarmsModel);
   }
 
   /** Returns a list of alarms created from a given rule */
   static getAlarmsForRule(id, params = {}) {
     return HttpClient.get(`${ENDPOINT}alarmsbyrule/${id}?${stringify(params)}`)
       .map(toAlarmsForRuleModel);
   }
+
+  /** Returns a telemetry events */
+  static getTelemetryByMessages(params = {}) {
+    return HttpClient.get(`${ENDPOINT}messages?${stringify(params)}`)
+      .map(toMessagesModel);
+  }
+
+  static getTelemetryByDeviceIdP1M(deviceIds = '') {
```

Comment:
What is P1m, PT1M, P15M, PT15M ? Can you add comments for these.

Reply:
These are time range constants. These will be documented later. The process I'm currently following is get the basic setup working per dashboard panel then I will be doing a unification pass (making use of duplicated calls etc. by moving the calls into the parent component or the redux store). So this is future work.